frontend/src/components/Summary/SummaryTable.jsx

```diff
@@ -57,15 +56,7 @@ const SummaryTable = ({ lines }) => {
       }
       let seq = element.seq;
       const caseImplementation = caseArray.find((each) => each.seq === seq);
-      if (
-        element.caught === true ||
-        element.caught === false ||
```

Comment:
Here we can't ignore the `caught` property as it also shows _**errored**_ cases.
To make the expression simple, maybe we can do something like `element.caught !== null`

Reply:
Hmm, I guess it's not as simple as I thought. Do you know what does `caught` signifies? When it's set to `true` and when to `false`?

Reply:
I am not 100% sure, but I guess when tests are running and get failed due to some reason, then the `caught` sets to `false`, and if some error occurred and the tests didn't run due to some reason, then the `caught` sets to `true`. 

I have gone through the data of all the drafts and built the logic based on that. I might have taken brute force approach, but I can surely say we do require both conditions to be considered (i.e. `caught` and `errored`). Skipping `caught === false` may not cause any issue in the latest draft, but it indeed misses out on some data in older drafts.(`draft3`, `draft4`)

However, @Julian, can you please confirm?

Reply:
Yeah, I looked briefly into the generated reports and now I kind of get it. Bottom line is that you're right and the `caught` field should be checked. And also this change is from previous PR and is already in main - gonna fix it with null check you suggested